src/vm_stack.zig

```diff
@@ -1002,7 +1003,7 @@ const InstructionFuncs = struct {
             while (i < array_len) : (i += 1) {
                 const value_start = i * byte_count;
                 const value_end = value_start + byte_count;
-                ret[i] = std.mem.readIntSliceLittle(read_type, mem[value_start..value_end]);
+                ret[i] = std.mem.readInt(read_type, mem[value_start..value_end][0..byte_count], .little);
```

Comment:
Can this be `std.mem.readInt(read_type, mem[value_start..value_end], .little);`?